packages/headless/src/chat.ts

```diff
@@ -122,11 +144,24 @@ type Action =
     }
   | {
       type: ActionType.ActivateMessage;
-      message: { refId: string; room: number };
+      message: IMessage.ClientSideMessage;
```

Comment:
`AttributedMessage` or `StatedMessage` is much better name. 

```suggestion
      message: IMessage.AttributedMessage;
```